plugins.xml

```diff
@@ -4531,6 +4531,101 @@
 			<institution>University of Pittsburgh</institution>
 			<email>uls-systemsdevelopment@pitt.edu</email>
 		</maintainer>
+		<release date="2022-04-29" version="1.1.0.1" md5="24f17491fad5ef933adfa677d22780cd">
+			<package>https://github.com/ulsdevteam/pkp-emailIssueToc/releases/download/v1.1.0-1/emailIssueToc-1.1.0-1.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.3.0.0</version>
+				<version>3.3.0.1</version>
+				<version>3.3.0.2</version>
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
+				<version>3.3.0.9</version>
+				<version>3.3.0.10</version>
+			</compatibility>
+			<compatibility application="omp">
```

Comment:
Please rescind the compatibility statement with OMP and OPS.  This plugin is specific to Journal Issue Table of Contents.